app/src/main/kotlin/codes/chrishorner/socketweather/data/ApiModels.kt

```diff
@@ -43,13 +45,18 @@ data class DateForecast(
     val short_text: String?,
     val icon_descriptor: String,
     val rain: Rain,
+    val uv: UV,
     val now: CurrentInformation? = null
 )
 
 data class Rain(val amount: Amount, val chance: Int) {
   data class Amount(val min: Float?, val max: Float?, val units: String)
 }
 
+data class Wind(val speed_knot: Int, val speed_kilometre: Int, val direction: String)
+
+data class UV(val max_index: Int?, val category: String?, val start_time: Instant?, val end_time: Instant?)
```

Comment:
Initially I was going to suggest making `category` an enum, but it looks like BOM seems to list it as `extreme`, or null 😛. Let's just remove the field for now.

Also `Uv` instead of `UV`.

```suggestion
data class Uv(val max_index: Int?, val start_time: Instant?, val end_time: Instant?)
```

Reply:
> Initially I was going to suggest making `category` an enum, but it looks like BOM seems to list it as `extreme`, or null 😛. Let's just remove the field for now.

Something, something, trees for the forest. I spent so much time looking at that data but never noticed. 🤦🏻